src/externalized/ItemModel.h

```diff
@@ -118,5 +120,5 @@ struct ItemModel
 	uint16_t   fFlags;
 
 	void serializeFlags(JsonObject &obj) const;
-	uint32_t deserializeFlags(JsonObject &obj) const;
+	static uint32_t deserializeFlags(JsonObject &obj);
```

Comment:
While you're fixing this oversight, please also change the return type to `uint16_t` (same as `ItemModels::fFlags`) and make obj a const reference.

Reply:
Fixed, thanks.